### PR TITLE
add list of newly added users to admin create user page

### PR DESCRIPTION
**Pull request by tesk9**
As admin, go to /#/admin/user/create

This gives you a form to create new users. All fields are required (email in form [a-z]+\@[a-z]+\.[a-z]+, roughly).
After adding a user, you should see a list appear/update with these new users. Their names link to their profile pages. The remove button should remove them from the page and make a delete request so that they are gone from the database too.

### CODE REVIEW

File: assets/js/templates/views/team.html [hunk 1]
```diff
@@ -37,16 +37,17 @@
...
-  <div class="col-xs-4">
-    <ul ng-repeat="user in team.users">
-      <li>
-        <a href="/#/user/{{user.id}}">{{user.name}}</a> 
-        <button type="button" class="btn btn-danger btn-xs" ng-click="remove(user.id)" ng-show="isAdmin">
-          <span class="glyphicon glyphicon-minus" aria-hidden="true"></span> Remove
-        </button>
-      </li>
-    </ul>
+  <br><br>
```
Comment (jplouie) [inline assets/js/templates/views/team.html]: ..

File: assets/js/templates/views/user_create.html [hunk 2]
```diff
@@ -26,4 +26,16 @@
       </form>
     </div>
   </div>
+  <div class="row row-bottom-margin">
```
Comment (jplouie) [inline assets/js/templates/views/user_create.html]: br

**Comment by kennyczadzeck** [comment]
Seems to work fine, but if I navigate away from list of newly added users and then come back, there is no way for me to get to the list of those created users. This is probably because they aren't associated to a team during this process. Not sure if that is intentional or not.

**Comment by tesk9** [comment]
It is intentional. We are going to need a separate view for all users, and we need a decision on how many viewers to show.

File: assets/js/controllers/user_creation_ctrl.js [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
   };
+
+  $scope.remove = function(userId) {
+    User.User.delete({id: userId});
+    $scope.users = $scope.users.filter(function(user) {
+      if(user.id !== userId) {
+        return user;
+      }
+    });
+  }
```
Comment (jplouie) [inline assets/js/controllers/user_creation_ctrl.js]: ;

**Pull request merged**